client/asset/interface.go

```diff
@@ -1141,6 +1182,23 @@ type BondTxInfo struct {
 	BondID dex.Bytes `json:"bondID"`
 }
 
+// NoCompletionRequiredBridgeTxID is a placeholder ID used when a bridge
+// transaction does not require a corresponding completion transaction. This
+// typically occurs when bridging from a base chain to a Layer 2, where the
+// network automatically allocates tokens to the recipient without a mint step.
+const NoCompletionRequiredBridgeTxID = "no-completion-required"
+
+// BridgeCounterpartTx identifies an initiation transaction if it is part of a
+// completion transaction, or a completion transaction if it is part of an
+// initiation transaction.
```

Comment:
I'm probably missing it, but where is the completion tx's `BridgeCounterpartTx` being added to the initiation tx `WalletTransaction`?

Reply:
It's being added in `bridgeManager.markBridgeComplete`.